oauthproxy.go

```diff
@@ -458,17 +458,19 @@ func (p *OAuthProxy) ErrorPage(rw http.ResponseWriter, code int, title string, m
 }
 
 //<!--COCO Begin-->
-// COCO: logout from cocostore
-//<!--COCO End-->
+// COCO: logout from coco
 func (p *OAuthProxy) CocostoreLogout(rw http.ResponseWriter, title string) {
         rw.WriteHeader(http.StatusOK)
+        // COCO: create a  stracture to pass a title in logout.html template
         t := struct {
                 Title       string
         }{
                 Title:       fmt.Sprintf("%d", title),
         }
+        // COCO: execute logout.html by passing stracture t
```

Comment:
- there is a typo here as well in structure

Reply:
handled